test/gateway/pollingInterval.js

```diff
@@ -119,9 +119,7 @@ test('Polling schemas', async (t) => {
   t.deepEqual(JSON.parse(res2.body), {
     errors: [
       {
-        message:
-          'Cannot query field "lastName" on type "User". Did you mean "name"?',
-        locations: [{ line: 6, column: 13 }]
```

Comment:
why are we losing `locations`? I think it was quite handy and needed.

Reply:
The locations are set by `errorHandler` and it is not called anymore, instead we call `errorFormatter` that calls `defaultErrorFormatter` in our tests. Maybe we can set locations on `defaultErrorFormatter`?

Reply:
I think should move the locations to the formatter.